src/main/java/core/basesyntax/Ball.java

```diff
@@ -0,0 +1,16 @@
+package core.basesyntax;
+
+public class Ball {
+    private final String color;
+    private final int number;
+
+    public Ball(String color, int number) {
+        this.color = color;
+        this.number = number;
+    }
+
+    @Override
+    public String toString() {
+        return "with color: " + color + " and number: " + number;
```

Comment:
make toString a little more informative: 
```suggestion
        return "The ball color is " + color + " and has a number " + number;
```